src/derived-states.js

```diff
@@ -8,69 +8,78 @@ See the accompanying LICENSE file for terms.
 // this will eventually move to context parser and it will not be a sparse matrix
 // Transition table based on https://html.spec.whatwg.org/multipage/syntax.html
 var DerivedState = {};
+
+/*
+// used to generate DerivedState.Transitions 
 DerivedState.TransitionsSparse = {
-    1:  {1: 1},
-    10: {1: 2, 43: 6},
-    13: {43: 6},
-    16: {43: 6},
-    19: {43: 6},
-    27: {43: 6},
-    34: {1: 2, 43: 6},
-    35: {1: 2, 36: 3, 43: 6},
-    36: {1: 2, 43: 6},
-    38: {42: 4},
-    39: {42: 4},
-    40: {1: 2, 34: 4},
-    42: {1: 2, 43: 6},
-    43: {1: 2},
+    1:  {1: 0x10},
```

Comment:
on a second thought, are you expecting something like 0x01 refers to the least significant bit (from right to left, first bit), 0x02 refers to the second, 0x04 refers to the third, and 0x08 refers to the fourth one, so and so forth?